ns1/rest/transport/requests.py

```diff
@@ -45,7 +45,7 @@ def _rateLimitHeaders(self, headers):
             "remaining": int(headers.get("X-RateLimit-Remaining", 100)),
```

Comment:
In my case, I have a response that is not JSON. Currently all responses are assumed to be JSON so I needed a way to handle this